### PR TITLE
vite-plugin: support Qwik

### PR DESCRIPTION
**Pull request by wmertens**
This adds the qwik plugin to the SSR list, the rest happens automatically.

### CODE REVIEW

**Comment by wmertens** [comment]
Just a note for anyone needing this before it's merged: You can set this process env in vite.config.js as a workaround, like so:

```js
// Workaround until https://github.com/vanilla-extract-css/vanilla-extract/pull/895 is accepted
process.env.VITE_RSC_BUILD = "true";

export default defineConfig(() => {
  const cfg = {
    plugins: [
      qwikCity(),
      qwikVite(),
      tsconfigPaths(),
      // This has to come after qwikVite, or the exports break
      vanillaExtractPlugin(),
    ],
  };
  return cfg;
});
```

**Comment by wmertens** [comment]
Just a note for anyone needing this before it's merged: You can set this process env in vite.config.js as a workaround, like so:

```js
// Workaround until https://github.com/vanilla-extract-css/vanilla-extract/pull/895 is accepted
process.env.VITE_RSC_BUILD = "true";

export default defineConfig(() => {
  const cfg = {
    plugins: [
      qwikCity(),
      qwikVite(),
      tsconfigPaths(),
      // This has to come after qwikVite, or the exports break
      vanillaExtractPlugin(),
    ],
  };
  return cfg;
});
```

File: .changeset/smart-dingos-search.md
Comment (mattcompiles) [inline .changeset/smart-dingos-search.md:5]:
```suggestion
Add support for Qwik
```

**Review by mattcompiles — APPROVED** [review]

**Review by mattcompiles — COMMENTED** [review]
Need to run `yarn format`, then it should be all good.

**Comment by wmertens** [comment]
@mattcompiles looks like it needs another approval to run the workflow :)

**Comment by n8sabes** [comment]
Hi @mattcompiles, I just answered #880 pointing them to the stackblitz workaround, and the Qwik community is continually asking about it on Discord.

As the Qwik community seems to be rapidly adopting VE, any idea when it might get pushed?

**Comment by mattcompiles** [comment]
@n8sabes releasing now mate.

**Comment by n8sabes** [comment]
@mattcompiles, Thank you. 🥳

**Pull request merged**
